### ISSUE TITLE
member yaml file format update

### ISSUE THREAD

**Issue by DenzilFrancisCrasta** [RBC-DSAI-IITM/rbc-dsai-iitm.github.io#4]
Since many members have multiple designations and research interests 
change designation and areas to lists in yaml 

designations:
  - Professor IITM
  - Visiting Professor MIT  

Same for areas 

areas:
  - Machine Learning
  - Pattern Recognition

**Comment by thulasri** [comment]
Sure will do

**Comment by thulasri** [comment]
done:)





Thanks & Regards
Thulasi

**Comment by karthikraman** [comment]
This seems to trigger a page build failure. Why can't we list the designations as an array? Like

`designation: ["Professor, IITM", "Visiting Professor, MIT"]`

?

**Comment by karthikraman** [comment]
The page build failed for the `master` branch with the following error:

There was a YAML syntax error on line 5 column 1 in `_members/raghur.yml`: `could not find expected ':' while scanning a simple key`.

**Comment by DenzilFrancisCrasta** [comment]
I have pushed the fix. Do report if the issue persists

**Comment by abhisheknaik96** [comment]
1. Why do we have files in ```_collaborator``` and ```_faculty``` folders?
2. The faculty.yml pages have a single attribute ```designation``` instead of a list ```designations```, causing the formatting to break for some faculty. I'll fix this.

**Comment by abhisheknaik96** [comment]
Fixed the faculty designation issue. 
@DenzilFrancisCrasta - Can you populate the Projects page with the appropriate css?

**Comment by thulasri** [comment]
shall i do the changes in yml file??

apologies for making wrong fix





Thanks & Regards
Thulasi

**Comment by karthikraman** [comment]
No, I wanted separate files for everything... I want fewer data files, and
separate yml files -- which literally get translated into html... In the
future, this will be peaceful for editing by one and all...
